Cargo.toml

```diff
@@ -2,21 +2,29 @@
 name = "darts"
 description = "A double array trie, A Forward Maximum Matching Searcher."
 repository = "https://github.com/andelf/rust-darts"
+documentation = "https://docs.rs/darts"
 version = "0.1.0"
 license = "MIT"
-authors = ["andelf <andelf@gmail.com>"]
+authors = ["andelf <andelf@gmail.com>",
+           "Paul Meng <me@paulme.ng>",
+           "messense <messense@icloud.com>"]
 readme = "README.md"
-keywords = ["trie", "darts"]
-categories = ["encoding"]
+keywords = ["trie", "darts", "string", "search", "text"]
+categories = ["data-structures", "text-processing"]
 edition = "2018"
+exclude = ["/benches", "/priv", "/.travis.yml"]
```

Comment:
`exclude = ["/benches/**", "/priv/**", "/.travis.yml"]` should work.

https://github.com/rust-lang/cargo/issues/4268